src/suricata.c

```diff
@@ -2369,6 +2369,7 @@ static int FinalizeRunMode(SCInstance *suri, char **argv)
 {
     switch (suri->run_mode) {
         case RUNMODE_UNKNOWN:
+            SCLogError("Error: Missing run mode. Please specify a capture runmode.");
```

Comment:
nit: there's no need to use `Error` in the message itself, for the usage of `SCLogError` will ensure that the log output will indicate that this is an error message.